setup.py

```diff
@@ -33,7 +33,8 @@ def get_version():
     platforms='any',
     install_requires=[
         'redis >= 2.7.0',
-        'click >= 5.0'
+        'click >= 5.0',
+        'redis-py-cluster'
```

Comment:
Most people don't need Redis cluster support so I think we should make this dependency optional.

Reply:
You mean remove redis-py-cluster? If that, I will add redis-py-cluster in travis config, because test cases need it.

Reply:
Yes, thank you.